webrtc.html

```diff
@@ -3780,6 +3780,7 @@
     readonly        attribute DOMString?              sdpMid;
     readonly        attribute unsigned short?         sdpMLineIndex;
     readonly        attribute DOMString?              foundation;
+    readonly        attribute byte?                   component;
```

Comment:
Sure; I guess it's impossible for it to be anything other than RTP or RTCP. Plus ORTC already has `RTCIceComponent`.